ci_framework/playbooks/01-bootstrap.yml

```diff
@@ -2,6 +2,15 @@
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: true
   tasks:
+    - name: Set custom ansible_user_dir reusable fact
+      tags:
+        - always
+      when:
+        - ansible_user_dir is not defined
+      ansible.builtin.set_fact:
+        ansible_user_dir: "{{ lookup('env', 'HOME') }}"
```

Comment:
lookup('env', 'XXX') looks up things on the deployment host [1].

If you want to get $HOME on the target host, you need to use "{{ ansible_env.HOME }}" 

This is a non issue when using the default cifmw_target_host since it is localhost, but it will become an issue when a different value is set. 

[1] https://docs.ansible.com/ansible/latest/collections/ansible/builtin/env_lookup.html
[2] https://stackoverflow.com/questions/57207045/ansible-lookup-not-working-for-enviroment-variables

Reply:
Indeed. It would work only thanks to the "hosts: localhost" usage we currently have. But since we're planing on consuming remote host, we'll hit this kind of issue sooner or later :(. `lookup` is local to the controller only, unfortunately.

Another way would be to ensure we gather minimal facts if this variable isn't set. Probably better imho. WDYT?